src/ferryd/core/repo.go

```diff
@@ -234,7 +234,6 @@ func (r *RepositoryManager) CreateRepo(db libdb.Database, id string) (*Repositor
 	return repository, nil
 }
 
-// DeleteRepo is not yet implemented
 func (r *RepositoryManager) DeleteRepo(db libdb.Database, pool *Pool, id string) error {
```

Comment:
This should still have a comment, albeit one a bit more useful than the previous one.

Reply:
Resolved by my commit.